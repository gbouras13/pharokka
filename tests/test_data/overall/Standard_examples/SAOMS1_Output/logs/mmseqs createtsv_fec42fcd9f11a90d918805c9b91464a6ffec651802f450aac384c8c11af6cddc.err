Command line: mmseqs createtsv ../../databases/vfdb SAOMS1_Output/VFDB_target_dir/target_seqs SAOMS1_Output/VFDB/results_mmseqs SAOMS1_Output/vfdb_results.tsv --full-header --threads 8
