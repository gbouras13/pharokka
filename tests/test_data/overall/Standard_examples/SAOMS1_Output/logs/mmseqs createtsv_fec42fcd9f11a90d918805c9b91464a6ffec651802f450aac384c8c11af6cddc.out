createtsv ../../databases/vfdb SAOMS1_Output/VFDB_target_dir/target_seqs SAOMS1_Output/VFDB/results_mmseqs SAOMS1_Output/vfdb_results.tsv --full-header --threads 8 

MMseqs Version:                 	13.45111
First sequence as representative	false
Target column                   	1
Add full header                 	true
Sequence source                 	0
Database output                 	false
Threads                         	8
Compressed                      	0
Verbosity                       	3

Time for merging to vfdb_results.tsv: 0h 0m 0s 12ms
Time for processing: 0h 0m 0s 35ms
